docs/index.html

```diff
@@ -22,6 +22,9 @@ <h2>Development Guides</h2>
   <ul>
     <li><a href="https://docs.google.com/document/d/1oKpvgstKlgmgUUja8hYqTqWxtwsgIbONoUaEj8lO0FE/edit">Embulk Input Plugin Development Guide</a></li>
   </ul>
+  <ul>
+    <li><a href="https://github.com/embulk/embulk/blob/master/docs/eeps/eep-0000.md">EEPs(Embulk Enhancement Proposals)</a></li>
+  </ul>
```

Comment:
* You may want to add this `<li>` block within the `<ul>` block above.
* Let's change the description to `Embulk Enhancement Proposals (EEPs)`

Reply:
Thanks! Fixed.